### PR TITLE
dns/bind: add support for transfer-source[-v6] options

### PR DESCRIPTION
**Pull request by fraenki**
...and it also fixes the indention of the surrounding options in named.conf (no-op change), because it looks better this way. :)

### CODE REVIEW

**Comment by fraenki** [comment]
@mimugmail please review.

File: dns/bind/src/opnsense/mvc/app/controllers/OPNsense/Bind/forms/general.xml [hunk 1]
```diff
@@ -33,6 +33,20 @@
         <type>text</type>
         <help>Set the port the service should listen to.</help>
     </field>
+    <field>
+        <id>general.transfersource</id>
+        <label>Transfer Source IP</label>
+        <type>text</type>
+        <advanced>true</advanced>
+        <help>This determines which local address is bound to IPv4 TCP connections used to fetch zones transferred inbound by the server.</help>
```
Comment (8191) [inline dns/bind/src/opnsense/mvc/app/controllers/OPNsense/Bind/forms/general.xml:41]: Even technically correct, the help text is quite hard to understand.

```suggestion
        <help>Specify the IPv4 address used as a source for zone transfers.</help>
```

File: dns/bind/src/opnsense/service/templates/OPNsense/Bind/named.conf [hunk 2]
```diff
@@ -15,16 +15,24 @@ options {
...
+{% endif -%}
+
+{% if helpers.exists('OPNsense.bind.general.transfersourcev6') and OPNsense.bind.general.transfersourcev6 != '' %}
+        transfer-source-v6 {{ OPNsense.bind.general.transfersourcev6 }};
+{% endif -%}
 
 {% if helpers.exists('OPNsense.bind.general.forwarders') and OPNsense.bind.general.forwarders != '' %}
         forwarders    { {{ OPNsense.bind.general.forwarders.replace(',', '; ') }}; };
-{% endif %}
+{% endif -%}
```
Comment (8191) [inline dns/bind/src/opnsense/service/templates/OPNsense/Bind/named.conf:35]: Is the dash before the ending `%}` intentional? Is there a special meaning for `-%}`?

Comment (fraenki) [reply dns/bind/src/opnsense/service/templates/OPNsense/Bind/named.conf:35]: It is used to remove obsolete whitespace, see [docs here](https://jinja.palletsprojects.com/en/2.10.x/templates/#whitespace-control).

File: dns/bind/src/opnsense/mvc/app/controllers/OPNsense/Bind/forms/general.xml (hunk 1 above)
Comment (fraenki) [reply dns/bind/src/opnsense/mvc/app/controllers/OPNsense/Bind/forms/general.xml]: It's the official description for this option, but I don't mind changing it.

**Review by mimugmail — APPROVED** [review]
Looks good, thx! :)

**Pull request merged**